include/xtypes/UnionType.hpp

```diff
@@ -157,6 +157,16 @@ class UnionType : public AggregationType
         return add_case_member(labels, Member(name, type), is_default);
     }
 
+    template<typename T>
+    UnionType& add_case_member(
+            const T& label,
+            const std::string& name,
+            const DynamicType& type,
+            bool is_default = false)
+    {
+        return add_case_member(std::vector<T>(1, label), Member(name, type), is_default);
+    }
+
```

Comment:
Is this method really useful?
I don't see any advantage to call it instead of `add_case_member({label}, Member(name, type), is_default);`